server/mailet/mock-smtp-server/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM openjdk:8u222-jre
+
+# Ports that are used:
+#
+# 25   SMTP without authentication
+# 8000 Web Admin interface
```

Comment:
That's not webadmin (that's confusing)

Prefer "HTTP interface for setting mock SMTP server behavior"